core/modules/field_ui/field_ui.api.php

```diff
@@ -99,16 +107,20 @@ function hook_field_instance_settings_form($field, $instance) {
  *   The field structure being configured.
  * @param $instance
  *   The instance structure being configured.
+ * @param $form
+ *   The (entire) configuration form array, which will usually have no use here.
+ * @param $form_state
+ *   The form state of the (entire) configuration form.
  *
  * @return
  *   The form definition for the widget settings.
  */
```

Comment:
We need a `@since` item added to the docblock. e.g:

```
@since 1.13.0 Added $form and $form_state parameters.
```

Reply:
Done! 😄

Reply:
@klonos you might need to add @since to each hook_ definition here.